### PR TITLE
Signup

### PR DESCRIPTION
**Pull request by yashodaAgarwal**
Created authentication (Signup and login ) features for the application.
Made cart and wishlist page private.
Plese reviw my code

### CODE REVIEW

File: src/pages/home-page/home-page.css [hunk 1]
```diff
@@ -17,19 +17,29 @@
...
 }
+.left-nav{
+  width: 50%;
+}
+.category-flex{
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  align-items: center;
+  row-gap: 70px;
```
Comment (Ananyamadhu08) [inline src/pages/home-page/home-page.css:28]: can use rem instead of px for higher values.

**Review by Ananyamadhu08 — APPROVED** [review]
Hey good job on the UI! Your code is clean and structured. You can use rem instead of px for higher values in your code, other than your code looks to go you can go ahead and merge.

File: src/pages/home-page/home-page.css (hunk 1 above)
Comment (yashodaAgarwal) [reply src/pages/home-page/home-page.css:28]: at some plces px is used intentionally to maintain the UI  for smaller screen

**Review by m26unkwn — APPROVED** [review]
code is clean and good.

uncomment all unnecessary code.

**Pull request merged**